ansible_ai_connect/main/settings/base.py

```diff
@@ -696,5 +696,15 @@ def is_ssl_enabled(value: str) -> bool:
 pipelines = [i for i in REGISTRY_ENTRY.keys() if issubclass(i, MetaData)]
 pipeline_config: dict = {k.__name__: ANSIBLE_AI_WCA_CONFIG for k in pipelines}
 
+# The ChatBot does not use the same configuration as everything else
+pipeline_config["ModelPipelineChatBot"] = {
+    "provider": "http",
+    "config": {
+        "inference_url": CHATBOT_URL or "http://localhost:8000",
+        "model_id": CHATBOT_DEFAULT_MODEL or "granite3-8b",
```

Comment:
Updated the default Model Id and removed `timeout`.

The `timeout` setting is still supported; but it is no longer set for ChatBot.